src/components/NewConnectionModal/index.tsx

```diff
@@ -111,11 +113,20 @@ const NewConnectionModal: React.FC<ModalProps> = ({
         const errors = getValidationErrors(err)
         formRef.current?.setErrors(errors)
       } else {
-        addToast({
-          type: 'error',
-          title: 'Error on connection',
-          description: 'Error estabilishing connection with your Redis server'
-        })
+        const message = err.message
+        if (message.search('WRONGPASS')) {
+          addToast({
+            type: 'error',
+            title: 'Error on connection',
+            description: 'Damn it, wrong password maybe 1234?'
```

Comment:
I think that `'Damn it, wrong password! Maybe 1234?'` would be better.

Reply:
Okay